CoreFoundation/PlugIn.subproj/CFBundle.c

```diff
@@ -951,7 +951,13 @@ CF_PRIVATE CFURLRef _CFBundleCopyAppStoreReceiptURLInDirectory(CFURLRef bundleUR
 CFURLRef _CFBundleCopyAppStoreReceiptURL(CFBundleRef bundle) {
     return _CFBundleCopyAppStoreReceiptURLInDirectory(bundle->_url, bundle->_version);
 }
-        
+
+#ifdef CF_OPEN_SOURCE
```

Comment:
Who defines `CF_OPEN_SOURCE`?

Reply:
That's a stray #ifdef; it'll be removed shortly.